main.go

```diff
@@ -75,11 +77,16 @@ func main() {
 	ctx := context.Background()
 	log := logger.NewLogger(ctx)
 	ctx = context.WithValue(ctx, logger.LoggerKey{}, log)
+	undo, maxprocsError := maxprocs.Set(maxprocs.Logger(stdlog.Printf))
+	defer undo()
 
-	if err := handle(ctx); err != nil {
+	if maxprocsError != nil {
+		stdlog.Fatalf("failed to set GOMAXPROCS: %v", maxprocsError)
+	} else if err := handle(ctx); err != nil {
```

Comment:
nit: no need for else if here

Reply:
To handle the maxprocs error this is needed right?

Reply:
I meant can just do:

```
if maxprocsError != nil {
  // some code
}

if err := handle(ctx); err != nil {
  // some code
}
```

Reply:
Cool, I'll update that